app/components/workflow_process_row.rb

```diff
@@ -4,7 +4,7 @@
 class WorkflowProcessRow < ApplicationComponent
   # @param [Dor::Workflow::Response::Process] process_status the model for the WorkflowProcess
   # @param [Integer] index the row index
-  # @param [Dor::Item] item the repository object that the workflow is about
+  # @param [Cocina::Models::DRO] item the repository object that the workflow is about
   def initialize(process:, index:, item:)
```

Comment:
I know it changes a public interface, but should `item` here change to `cocina_item` per our new cocina-models usage guidance?